lib/rock.rb

```diff
@@ -0,0 +1,14 @@
+class Rock
+  attr_reader :value
+  def initialize
```

Comment:
Would it work with a generic 'shape/choice' class where the value is passed on initialisation? This would simplify the code base a little.